src/ClockCountDown.elm

```diff
@@ -0,0 +1,102 @@
+module ClockCountDown exposing (Model, Msg(..), init, main, subscriptions, update, view)
+
+import Browser
+import Html exposing (..)
+import Task
+import Time
+
+
+main =
+    Browser.element
+        { init = init
+        , view = view
+        , update = update
+        , subscriptions = subscriptions
+        }
+
+
+type alias Model =
+    { zone : Time.Zone
+    , time : Time.Posix
+    }
+
+
+type Msg
+    = Tick Time.Posix
+    | AdjustTimeZone Time.Zone
+    | CurrTime Time.Posix
+
+
+init : () -> ( Model, Cmd Msg )
+init _ =
+    ( Model Time.utc (Time.millisToPosix 0)
+    , Cmd.batch
+        [ Task.perform AdjustTimeZone Time.here
+        , Task.perform CurrTime Time.now
+        ]
+    )
+
+
+update : Msg -> Model -> ( Model, Cmd Msg )
```

Comment:
Just like we don't want `init` to consume an unused argument, `update` shouldn't return a `Cmd Msg` if it always returns the same value.
Thus:
```elm
update : Msg -> Model -> Model
```